subsys/bluetooth/audio/bap_broadcast_assistant.c

```diff
@@ -34,17 +35,21 @@ LOG_MODULE_REGISTER(bt_bap_broadcast_assistant, CONFIG_BT_BAP_BROADCAST_ASSISTAN
 
 #define MINIMUM_RECV_STATE_LEN          15
 
+struct bap_broadcast_assistant_recv_state_info {
+	uint8_t src_id;
+	bt_addr_le_t addr;
+	uint8_t adv_sid;
+	uint32_t broadcast_id;
+
+	/** Cached PAST available */
+	bool past_avail;
```

Comment:
```suggestion
	uint8_t src_id;

	/** Cached PAST available */
	bool past_avail;
	uint8_t adv_sid;
	uint32_t broadcast_id;
	bt_addr_le_t addr;
```

Suggest a reordering to put the 3 1-octet values in the same word

Reply:
Done